src/terminal/osc.zig

```diff
@@ -1662,6 +1681,19 @@ test "OSC: show desktop notification with title" {
     try testing.expectEqualStrings(cmd.show_desktop_notification.body, "Body");
 }
 
+test "OSC: OSC9;2 conemu message box" {
```

Comment:
Can you add the following tests:

- Blank message
- Spaces only message
- No trailing semicolon `OSC 9;2`